src/blocks/marker/edit.js

```diff
@@ -100,21 +104,23 @@ export default function Edit( { attributes, setAttributes } ) {
 												{
 													aspectRatio: '4/3',
 													className:
-														'is-style-default map-info__image',
-													id: 74106,
+														'is-style-default carousel-slide__image',
+													id: 74197,
 													lightbox: {
 														aspectRatio: '4/3',
 														className:
-															'is-style-default map-info__image',
+															'is-style-default carousel-slide__image',
 														enabled: false,
-														id: 74106,
+														id: 74197,
 														linkDestination: 'none',
 														scale: 'cover',
 														sizeSlug: 'full',
+														url: '/wp-content/uploads/2024/01/Wikimedia_Foundation_AI_Blog_Series_Header.png',
```

Comment:
Heads up that this URL (and the post ID) won't work on the Endowment site, but since it's just a fallback that's probably OK.

For placeholders I've just been using URLs from https://commons.wikimedia.org, which might work as placeholders. The attribution doesn't come through, so they can't be used as is. But maybe that would still be better than a broken image on the endowment sites?